tapdb/universe_perf_test.go

```diff
@@ -139,6 +155,52 @@ const (
 		SELECT asset_id, supply, COUNT(*) as num_leaves
 		FROM asset_supply
 		GROUP BY asset_id, supply`
+
+	// queryAggregatedStats gets aggregated stats for the universe. This
+	// corresponds to the QueryUniverseStats query.
+	queryAggregatedStats = `
+		WITH stats AS (
```

Comment:
why re-write this query? can't we grab it from some sqlc artifact?

Reply:
Hmm, good idea. Turns out we actually can.